CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # Changelog
 
+## 💬 1.7.0
+
+- ### Features
+
+  - **Reformat config api-key output - [bradyjoslin], [issue/889] [pull/910]**
+
+    We care a lot about our error output. This PR from community contributor @bradyjoslin pretties up the output from `wrangler config` to match when you pass the `--api-key` flag. Thanks for the PR @bradyjoslin!
+
+    [bradyjoslin]: https://github.com/bradyjoslin
+    [pull/910]: https://github.com/cloudflare/wrangler/pull/910
+    [issue/889]: https://github.com/cloudflare/wrangler/issues/889
+
+  - **Improve error message for `wrangler init --site` when wrangler.toml already exists - [ashleygwilliams], [issue/648] [pull/931]**
+
+    `wrangler init` generally expects that you don't already have a wrangler.toml present; however it is common that users want to add static site functionality to their existing wrangler project and will try using `wrangler init` to do so. Rather than simply complaining that the toml already exists, this PR adds the workers-site directory to the project, and prints out the suggested configuration to add to `wrangler.toml`. Much nicer!
```

Comment:
```suggestion
    `wrangler init` generally expects that you don't already have a `wrangler.toml` present; however it is common that users want to add static site functionality to their existing wrangler project and will try using `wrangler init` to do so. Rather than simply complaining that the toml already exists, this PR adds the workers-site directory to the project, and prints out the suggested configuration to add to `wrangler.toml`. Much nicer!
```